projects/lib/src/dynamic-views/item/item.component.html

```diff
@@ -5,21 +5,20 @@
 
 <ng-container
   *ngIf="!config.type.startsWith('_') && condition"
-  [ngTemplateOutlet]="templates[config.type || id]?.template"
+  [ngTemplateOutlet]="zoneRef.templates[config.type || id]?.templateRef"
   [ngTemplateOutletContext]="{ $implicit: config, data: _data, id: id }"
 ></ng-container>
 
-<ng-container *ngIf="config.type === '_container' && condition && !configurable">
-  <div *ngFor="let item of config.items; let index = index"
-    [uib-item]="item"
-    [templates]="templates"
+<ng-container *ngIf="config.type === '_container' && !configurable">
+  <ng-container *ngFor="let item of config.items; let index = index;">
+    <div *ngIf="item && condition"
```

Comment:
Why move `condition` inside *ngFor ? The condition applies for the whole container, so it should be where it was before. Also, why test `*ngIf="item"`? you should always have an item in `config.item`, as you cannot have a null or undefined in the container's configuration.